RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_tools.py

```diff
@@ -0,0 +1,108 @@
+import FWCore.ParameterSet.Config as cms
+
+def calibrateReducedEgamma(process):
+    process.load("EgammaAnalysis.ElectronTools.calibratedElectronsRun2_cfi")
+    process.load("EgammaAnalysis.ElectronTools.calibratedPhotonsRun2_cfi")
+    process.calibratedPhotons.produceCalibratedPhos = cms.bool(False)
+    process.calibratedElectrons.produceCalibratedEles = cms.bool(False)
+    process.slimmingTask.add(process.calibratedPhotons)
+    process.slimmingTask.add(process.calibratedElectrons)
+    process.reducedEgamma.applyPhotonCalibOnData = cms.bool(True)
+    process.reducedEgamma.applyPhotonCalibOnMC = cms.bool(True)
+    process.reducedEgamma.applyGsfElectronCalibOnData = cms.bool(True)
+    process.reducedEgamma.applyGsfElectronCalibOnMC = cms.bool(True)
+    process.reducedEgamma.photonCalibEnergySource = cms.InputTag("calibratedPhotons","EGMecalEnergy")
+    process.reducedEgamma.photonCalibEnergyErrSource = cms.InputTag("calibratedPhotons","EGMecalEnergyErr")
+    process.reducedEgamma.gsfElectronCalibEnergySource = cms.InputTag("calibratedElectrons","EGMecalTrkEnergy")
+    process.reducedEgamma.gsfElectronCalibEnergyErrSource = cms.InputTag("calibratedElectrons","EGMecalTrkEnergyErr")
+
+    process.reducedEgamma.gsfElectronCalibEcalEnergySource = cms.InputTag("calibratedElectrons","EGMecalEnergy")
+    process.reducedEgamma.gsfElectronCalibEcalEnergyErrSource = cms.InputTag("calibratedElectrons","EGMecalEnergyErr")
+
+    process.reducedEgamma.gsfElectronPFClusterIsoSources.append(cms.InputTag("calibratedElectrons","EGMscaleUpUncertainty"))
```

Comment:
this looks like a not appropriate overload of the meaning of the parameter gsfElectronPFClusterIsoSources

Reply:
if I'm not mistaken the full list of input tags with instance names (EGMscaleUpUncertainty etc)  used here is already available in other py files and can be imported directly. This will reduce copy-paste and make this code more maintainable.